src/Silakov.Homework8/Shop.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Silakov.Homework8
+{
+    internal class Shop
+    {
+        private PeopleGenerator _peopleGenerator;
+        private Queue<Person> _processingQueue;
+        private List<Thread> _processors;
+        private Thread _queueProcessingThread;
+
+        private static object locker = new object();
+        private static ThreadLocal<Random> _random;
+
+        private bool _isOpen;
+        private DateTime _storeClosingTime;
+
+        public Shop(PeopleGenerator peopleGenerator, int cashierNumber, int _shopOpeningHours)
```

Comment:
let's use the naming convention where parameter names start with lowercase character

Reply:
understood, thanks